mk/config.mk

```diff
@@ -29,7 +29,7 @@ CC_AR  := $(CROSS_PREFIX)$(CC_AR)
 #################
 # Common config #
 #################
-CFLAGS := \
+CFLAGS += \
```

Comment:
Unfortunately we can't do that without reverting further changes: We want user specified CFLAGS do overwrite default CFLAGS, but with this version it's the other way around